validate.go

```diff
@@ -163,6 +163,8 @@ func checkLinux(spec rspec.Linux, rootfs string) {
 	for index := 0; index < len(spec.Namespaces); index++ {
 		if !namespaceValid(spec.Namespaces[index]) {
 			logrus.Fatalf("Namespace %s is invalid.", spec.Namespaces[index])
+		} else if spec.Namespaces[index].Type == rspec.UTSNamespace && hostname != "" {
```

Comment:
Yes, `when hostname is not empty`.
@Mashimiao 's first commit did the right thing and I mislead him to squash code lines.